com.zen.browser.flatpak.yml

```diff
@@ -0,0 +1,66 @@
+app-id: com.zen.browser
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.mozilla.firefox.BaseApp
+base-version: '22.08'
+command: launch-script.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pcsc
+  - --socket=cups
+  - --persist=.zen
+  - --filesystem=xdg-download:rw
+  - --device=all
+  - --talk-name=org.freedesktop.FileManager1
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.a11y.Bus
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=com.zen.browser.*
+modules:
+  - name: zen_browser
+    buildsystem: simple
+    build-commands:
+      - tar -xvf zen.linux.tar.bz2
```

Comment:
type: archive will extract it automatically. Drop this and move what you need to `/app`.

Reply:
Move `zen/` into `/app`?

Reply:
Yea, I haven't checked the archive. Move what you need.